docs/style.md

```diff
@@ -2,45 +2,53 @@
 
 This is a set of recommended InSpec rules you should use when writing controls.
 
-## Control files
+# Control Files
 
-### 1. All controls should be located in the "controls" directory and end in ".rb"
+## Place control files in "controls/" and end them with ".rb"
 
-Reason: Most syntax highlighters will render InSpec files correctly across a wide list of tools.
+Most syntax highlighters will render InSpec files correctly across a wide list
+of tools.
 
-Avoid: `controls/ssh_config`
-Use: `controls/ssh_config.rb`
+Avoid: `controls/ssh_config`</br>
```

Comment:
Throughout the file, this PR adds HTML breaks in lists.  I think it would look better, and be more semantic, if the lists used a list marker:

* Avoid: `controls/ssh_config`
* Use: `controls/ssh_config.rb`

We could add levels, as well:

* Avoid: `controls/ssh_config`
  * Use: `controls/ssh_config.rb`

Reply:
I'm not a fan of levels in this case...but I like it better than `</br>` and have no other solutions.

I'll change them 😄

Reply:
Actually, how about the following since it works better with multiple avoids?:

Avoid: 
  - controls/ssh_config

Use: 
  - controls/ssh_config.rb

Reply:
I like that, too!